src/components/Card/Card.tsx

```diff
@@ -90,13 +88,12 @@ function Card({
 
   return (
     <Link
-      to={url ?? ''}
+      to={url}
       className={cardClassName}
-      onClick={onClick}
+      onClick={disabled ? (e) => e.preventDefault() : undefined}
       onMouseEnter={onHover}
       tabIndex={disabled ? -1 : 0}
-      onKeyDown={(event: KeyboardEvent) => (event.key === 'Enter' || event.key === ' ') && !disabled && onClick && onClick()}
-      role="button"
+      onKeyDown={(event: KeyboardEvent) => (event.key === 'Enter' || event.key === ' ') && !disabled}
```

Comment:
This method isn't doing anything. I think you can remove it. The disabled links won't be tabbable anyhow.